### PR TITLE
4657 - adding validation to the statistics

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-client/src/views/StartCaseInternal/StatisticsForm.jsx [hunk 1]
```diff
@@ -24,13 +24,7 @@ export const StatisticsForm = connect(
...
-      <FormGroup
-        errorText={
-          validationErrors.statistics &&
-          validationErrors.statistics[index] &&
-          validationErrors.statistics[index].deficiencyAmount
-        }
-      >
+      <>
         <label className="usa-label" htmlFor={`deficiency-amount-${index}`}>
           Decifiency
```
Comment (kkoskelin) [inline web-client/src/views/StartCaseInternal/StatisticsForm.jsx]: Typo here, should be "Deficiency"

**Review by kkoskelin — APPROVED** [review]
Just fix that one typo and good to go.

File: shared/src/business/entities/cases/Case.js [hunk 2]
```diff
@@ -661,11 +661,28 @@ Case.validationRules = {
...
-    .items(joi.object().meta({ entityName: 'Statistic' }))
-    .required()
-    .description('List of Statistic Entities for the case.'),
+  statistics: joi.when('hasVerifiedIrsNotice', {
+    is: true,
+    otherwise: joi
+      .array()
+      .items(joi.object().meta({ entityName: 'Statistic' }))
+      .optional()
+      .description('List of Statistic Entities for the case.'),
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.js]: Are you able to put this `.description` line after the `joi.when` so it doesn't have to be repeated multiple times?

File: web-client/src/views/StartCaseInternal/IRSNotice.jsx [hunk 3]
```diff
@@ -48,6 +50,7 @@ export const IRSNotice = connect(
                   key: e.target.name,
                   value: true,
                 });
+                refreshStatisticsSequence();
```
Comment (rachaelparris) [inline web-client/src/views/StartCaseInternal/IRSNotice.jsx:53]: I think you'll want to add this to the case type dropdown too, and the No option for IRS notice.

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**